packages/types/src/product/common.ts

```diff
@@ -245,6 +245,10 @@ export interface CreateProductDTO {
   metadata?: Record<string, unknown>
 }
 
+export interface UpdateProductDTO extends CreateProductDTO {
```

Comment:
todo: Not sure this is totally right, for example, the options DTO of the create is missing the id that can be provided, maybe we need to re check. Also, in those cases I would argue that we can have duplication and it is not a problem, but the idea is that both create and update can evolve independently

Reply:
Makes sense. I'll split it up.